explorer/src/constants/columns/portal.tsx

```diff
@@ -55,3 +56,29 @@ export const columns = ({ chain }: { chain: Chain }): ColumnDef<Portal>[] => [
     },
   },
 ];
+
+export const portalColumnsOption = {
+  name: {
+    width: "25%",
+  },
+  description: {
+    width: "50%",
+  },
+  owner: {
+    width: "25%",
+  },
+};
+
+export const skeletonPortals = (count: number) =>
+  Array(count)
+    .fill(null)
+    .map((_, index) => ({
+      id: `0x${index.toString(16).padStart(40, "0")}` as `0x${string}`,
+      name: EMPTY_STRING,
+      description: EMPTY_STRING,
+      ownerAddress: `${EMPTY_0X_STRING}${"0".repeat(40)}` as `0x${string}`,
+      ownerName: EMPTY_STRING,
+      modules: [] as `0x${string}`[],
```

Comment:
Probably not necessary to cast the empty Array?

Reply:
done